public/app/plugins/datasource/elasticsearch/datasource.js

```diff
@@ -331,6 +331,7 @@ function (angular, _, moment, kbn, ElasticQueryBuilder, IndexPattern, ElasticRes
     this.metricFindQuery = function(query) {
       query = angular.fromJson(query);
       query.query = templateSrv.replace(query.query || '*', {}, 'lucene');
+      query.field = templateSrv.replace(query.field || '*', {}, 'lucene');
```

Comment:
not all elasticsearch template queries have a field property. Need to check if the query has a field. Also the fallback to a wildcard does not make sense for the field.